test/User-test.js

```diff
@@ -0,0 +1,43 @@
+const chai = require("chai");
+const expect = chai.expect;
+const User = require("../src/User")
+const UserRepository = require("../src/UserRepository")
+
+describe("User", () => {
+  beforeEach(() =>{
+    user1 =  {
+                "id": 1,
+                "name": "Luisa Hane",
+                "address": "15195 Nakia Tunnel, Erdmanport VA 19901-1697",
+                "email": "Diana.Hayes1@hotmail.com",
+                "strideLength": 4.3,
+                "dailyStepGoal": 10000,
+                "friends": [
+                  16,
+                  4,
+                  8
+                ]
+              },
+    userRepository = new UserRepository([user1,user2,user3]);
```

Comment:
We are passing in`user2` and `user3` even though we only assigned value to user1. Maybe refine this?